jsdoc.json

```diff
@@ -3,13 +3,14 @@
     "include": [
       "packages/modeling/src/colors",
       "packages/modeling/src/connectors",
-      "packages/modeling/src/geometry",
-      "packages/modeling/src/geometry/geom2",
-      "packages/modeling/src/geometry/geom3",
-      "packages/modeling/src/geometry/path2",
-      "packages/modeling/src/geometry/poly2",
-      "packages/modeling/src/geometry/poly3",
+      "packages/modeling/src/geometries",
```

Comment:
Please revert as these changes (fixes for documentation) should be part of another PR.

Reply:
The latest change looks good. did you try to generate documentation?

`npm run docs`